chain/data.go

```diff
@@ -20,11 +20,16 @@
 
 package chain
 
-import "github.com/berachain/beacon-kit/primitives/common"
+import (
+	"github.com/berachain/beacon-kit/consensus/cometbft/service/delay"
+	"github.com/berachain/beacon-kit/primitives/common"
+)
 
 // SpecData is the underlying data structure for chain-specific parameters. All fields with a
 // `mapstructure` tag are required.
 type SpecData struct {
+	delay.Config `mapstructure:"block-delay-configuration"`
```

Comment:
doesn't feel correct that the cometbft/service config is imported here and in the chain spec. Can we move this config to the comet app config?

Reply:
I see that stylistically this config is handled differently from the others, still I think this arrangement is the best:

- embedding `delay.Config` here is just a way to cut down code duplication.
- This is not really a cometBFT only config (which is actually only parsed by CometBFT code). The majority logic of next block delay is an app thing and only one of the activation heights is strictly speaking a consensus quantity.